### PR TITLE
Removed unit fields and changed parameters json file for the module.

### PR DESCRIPTION
**Pull request by scaramonche**
Since the mass tolerance values were not used without the units, I just kept the values as they are (without splitting the values and the units into separate variables).

### CODE REVIEW

**Review by mlocardpaulet — APPROVED** [review]
looks good to me

File: proteobench/modules/dda_quant/plot.py
[the remark this answers is not in the document]
Comment (scaramonche) [reply proteobench/modules/dda_quant/plot.py:1]: Sure! Do you mean I should remove the two lines that show the tolerance information?

[the remark this answers is not in the document]
Comment (mlocardpaulet) [reply proteobench/modules/dda_quant/plot.py:1]: Keep everything. If anybody complains we can choose to remove information at a later stage.

[the remark this answers is not in the document]
Comment (scaramonche) [reply proteobench/modules/dda_quant/plot.py:1]: Thanks for checking it!

**Pull request merged**